### PR TITLE
Move NX-OS state tests to pytest

### PR DESCRIPTION
**Pull request by krionbsd**
### What does this PR do?
Move NX-OS state tests to pytest

### CODE REVIEW

**Review by twangboy — DISMISSED** [review]

File: tests/pytests/unit/states/test_nxos.py [hunk 1]
```diff
@@ -0,0 +1,549 @@
...
+
+@pytest.fixture
+def configure_loader_modules():
+    return {nxos_state: {}}
+
+
+def test_user_present_create():
+
+    """ UT: nxos module:user_present method - create """
+
```
Comment (s0undt3ch) [inline tests/pytests/unit/states/test_nxos.py:32]:
```suggestion
def test_user_present_create():
    """
    user_present method - create
    """

```

Comment (s0undt3ch) [reply tests/pytests/unit/states/test_nxos.py:32]: And the same approach for the other docstrings please.

**Review by s0undt3ch — CHANGES_REQUESTED** [review]

File: tests/pytests/unit/states/test_nxos.py (hunk 1 above)
Comment (krionbsd) [reply tests/pytests/unit/states/test_nxos.py]: Fixed

File: tests/pytests/unit/states/test_nxos.py [hunk 2]
```diff
@@ -0,0 +1,585 @@
...
+@pytest.fixture
+def configure_loader_modules():
+    return {nxos_state: {}}
+
+
+def test_user_present_create():
+
+    """
+    user_present method - create
+    """
```
Comment (s0undt3ch) [inline tests/pytests/unit/states/test_nxos.py:33]:
```suggestion
def test_user_present_create():
    """
    user_present method - create
    """
```

Comment (s0undt3ch) [reply tests/pytests/unit/states/test_nxos.py:33]: And the same for the other docstrings

**Review by s0undt3ch — CHANGES_REQUESTED** [review]

File: tests/pytests/unit/states/test_nxos.py (hunk 2 above)
Comment (krionbsd) [reply tests/pytests/unit/states/test_nxos.py]: Fixed

**Review by s0undt3ch — APPROVED** [review]

**Review by s0undt3ch — APPROVED** [review]

**Pull request merged**